README.md

```diff
@@ -22,25 +22,26 @@ exercises in order to grade them. **This script assists the relevant steps in th
 ## First steps
 1. Add your ArTEMiS credentials (username and password) into `config.yml`
```

Comment:
Add the `general` field here if needed!

Reply:
Documenting this publicly isn't necessary at this point as "fix_eclipse_imports" is set by default and most users won't care to change this.